app/src/main/res/layout/play_chapter_view.xml

```diff
@@ -8,26 +8,33 @@
 
     <import type="org.oppia.app.model.ChapterPlayState" />
 
+    <variable
+      name="index"
+      type="Integer" />
+
     <variable
       name="chapterSummary"
       type="org.oppia.app.model.ChapterSummary" />
   </data>
 
   <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/chapter_container"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_margin="4dp">
```

Comment:
Please add top and bottom padding.
![Screenshot_20191113-220806](https://user-images.githubusercontent.com/54615666/68784358-6cca9980-0662-11ea-8cc8-b2e2218ef989.png)
or just fix this spacing issue.